lib/src/main/java/growthbook/sdk/java/Experiment.java

```diff
@@ -92,40 +109,7 @@ public String toString() {
      * @return JsonElement
      */
     public static <ValueType> JsonElement getJson(Experiment<ValueType> object) {
-        JsonObject json = new JsonObject();
-
-        json.addProperty("key", object.getKey());
-
-        JsonElement variationsElement = GrowthBookJsonUtils.getJsonElementForObject(object.getVariations());
-        json.add("variations", variationsElement);
-
-        JsonElement weightsElement = GrowthBookJsonUtils.getJsonElementForObject(object.getWeights());
-        json.add("weights", weightsElement);
-
-        json.addProperty("isActive", object.getIsActive());
-        json.addProperty("coverage", object.getCoverage());
-
-        JsonElement namespaceElement = GrowthBookJsonUtils.getJsonElementForObject(object.getNamespace());
-        json.add("namespace", namespaceElement);
-
-        json.addProperty("force", object.getForce());
-        json.addProperty("hashAttribute", object.getHashAttribute());
-
-        return json;
-    }
-
-    /**
-     * A Gson serializer for {@link Experiment}
-     * @return a serializer for {@link Experiment}
-     * @param <ValueType> value type for the Experiment
-     */
-    public static <ValueType> JsonSerializer<Experiment<ValueType>> getSerializer() {
-        return new JsonSerializer<Experiment<ValueType>>() {
-            @Override
-            public JsonElement serialize(Experiment<ValueType> src, Type typeOfSrc, JsonSerializationContext context) {
-                return Experiment.getJson(src);
-            }
-        };
```

Comment:
Removing the custom serialization results in the `condition` field not being present and instead `conditionJson` on the serialized JSON but we don't need that to be a specific shape for any reason, so it's fine. Removing the custom serialization, while resulting in less-than-perfect JSON output, makes it easier to maintain.